src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -7,6 +7,8 @@
     <NoWarn>1718</NoWarn>
     <DefineConstants Condition="'$(TargetGroup)'=='netcoreapp'">$(DefineConstants);netcoreapp</DefineConstants>
     <DefineConstants Condition="'$(TargetGroup)'=='uapaot'">$(DefineConstants);uapaot</DefineConstants>
+    <!-- Please don't delete. We need App.config for netfx -->
+    <NETFxTestRunnerAppConfig>$(MSBuildProjectDirectory)\App.config</NETFxTestRunnerAppConfig>
```

Comment:
Does this have any impact when not running on netfx? Should we add a condition in here so that this is only included when building the netstandard configuration?